author/how-to/vscode.ftd

```diff
@@ -7,18 +7,18 @@ but we can install it ourselves.
 
 Follow these steps to install the "fastn-syntax" extension:
 
-1. Download the extension file from the following link: [fastn-syntax](https://github.com/harshdoesdev/fastn-syntax/blob/main/fastn-syntax-0.0.3.vsix).
+1. Download the extension file from the following link: [fastn-vscode](https://github.com/fastn-stack/fastn-vscode).
```

Comment:
Make this full download url so if user clicks on it the extension gets downloaded directly. Right now user will have to click on link and then search for another link and click there.